packages/pyright-internal/src/analyzer/importResolver.ts

```diff
@@ -8,6 +8,8 @@
  * runtime rules of Python.
  */
 
+import * as fs from 'fs';
```

Comment:
This is a nit, but just to keep it clear that this file doesn't actually access the filesystem directly, can you do a `import type { Dirent } from 'fs'` or similar?

Reply:
`fs` is also a variable used in a few places, so this may get shadowed.